tests/test_command.py

```diff
@@ -37,6 +37,60 @@ def execute_command(self, command: dict, response_type: str) -> dict:
         return result["content"]
 
 
+class TestLoadColumnsWithAggregatorCommand(TestCommand):
```

Comment:
Thanks for adding a test. Since all of the tests use the same columns and command, maybe you can include a `setUp` method to create them there once. Also, maybe run once without a query and once with, with a query that uses columns that you are not aggregating over, that way we can verify that the `query` method is working as expected.